index.Rmd

```diff
@@ -19,7 +19,7 @@ a:visited {color: #91170a;}
 
 # [WUR Geoscripting](https://geoscripting-wur.github.io/) <img src="https://www.wur.nl/upload/854757ab-168f-46d7-b415-f8b501eebaa5_WUR_RGB_standard_2021-site.svg" alt="WUR logo" style="height: 35px;"/>
 
-# Week 1, Tutorial 3: Intro to functions and refresher on R
```

Comment:
Removed the tutorial numbering to make it more flexible. Also, the tutorial could be adapted it a bit since this is no longer a "refresher" on R. I didn't want to change it in case it created problems elsewhere. Suggestion: "Intro to functions and handling spatial data in R".